README.md

```diff
@@ -61,10 +61,12 @@ If compiling the PPAPI (Chromium) plugin is enabled (on by default), keep in min
 There are two ways of building Lightspark. You can use the included script, by running `./build.sh`. or for a debug build, run `./build.sh -d`. You can also do it manually, with the following commands:
 
 ```bash
+git clone https://github.com/microsoft/vcpkg
+./vcpkg/vcpkg integrate install
```

Comment:
I think this should better be moved as an alternative to the chapter about installing the development packages for the various distributions.

Reply:
Yeah, I was using a code space, so it was almost impossible to do it any other way, but that should work, I don't know much about bash as a language, so you would have to come up with that, but I think it's certainly a good idea.